templates/terraform/expand_property_method.erb

```diff
@@ -160,4 +155,12 @@ func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d T
   // TODO: Property '<%= property.name -%>' of type <%= property.class -%> is not supported
 }
 <%   end # tf_types.include?(property.class) -%>
+
+<% if property.nested_properties? -%>
```

Comment:
This is a lot of Ruby grouped together- should we make this a contiguous code block?

Reply:
I'm all for it, but it turns out getting the two different erb injection types (`<%` and `<%=`) to play nicely in 1 block is much harder than I imagined.